cms/startup.py

```diff
@@ -9,6 +9,8 @@
 
 from django_startup import autostartup
 from monkey_patch import django_utils_translation
+from util import keyword_substitution
+from lms.startup import get_keyword_function_map
```

Comment:
I know that importing from CMS/LMS into common is forbidden. 

What's the policy w/r/t importing from LMS->CMS (and vice-versa)?

Ideally, the `get_keyword_function_map` function should be stored in a `common` location. But, I'm assuming that this method already exists upstream? Assuming so, let's defer and not diverge more than necessary.